android/third_party_libs.md

```diff
@@ -10,13 +10,13 @@ Source: <https://github.com/square/retrofit>
 Overview of OkHttp: <http://square.github.io/okhttp> <br/>
 Source for OkHttp: <https://github.com/square/okhttp>
 
-### Otto
+### EventBus
 
-An event bus tailored to Android, Otto supports communication between distinct application components, when the use of a traditional interface is inadequate or cumbersome. 
+An event bus tailored to Android, greenrobot's EventBus supports communication between distinct application components; generally, the event bus should only be used when a traditional interface is inadequate or cumbersome, or an event is broadcast to multiple listeners at once.  
```

Comment:
We should remove`generally, the event bus should only be used when a traditional interface is inadequate or cumbersome`, event bus shouldn't be used simply as way of avoiding passing callbacks